rosbag2_test_common/include/rosbag2_test_common/publisher_manager.hpp

```diff
@@ -41,9 +41,6 @@ class PublisherManager
     auto publisher_node = std::make_shared<rclcpp::Node>("publisher" + std::to_string(counter++));
     auto publisher = publisher_node->create_publisher<T>(topic_name);
 
-    // We need to publish one message to set up the topic for discovery
-    publisher->publish(message);
```

Comment:
is this just legacy or why is this code removed?

Reply:
With auto discovery we no longer need to advertise the topic before starting rosbag in the test.